src/library/basetracktablemodel.h

```diff
@@ -21,6 +30,10 @@ class BaseTrackTableModel : public QAbstractTableModel, public TrackModel {
             const char* settingsNamespace);
     ~BaseTrackTableModel() override = default;
 
+    QVariant getFieldVariant(const QModelIndex& index, ColumnCache::Column column) const;
+    QVariant getFieldVariant(const QModelIndex& index, const QString& fieldName) const;
```

Comment:
any chance of removing this overload too?

Reply:
Unfortunatly not. This is used by https://github.com/daschuer/mixxx/blob/eb38c67c0b17d6b0677f9f9050b4f46c94b83184/src/library/basesqltablemodel.cpp#L787
Which can point to diffrent ColumnCache::Column enums. 
(Of cause it is, but not with a bigger refactoring)

Reply:
Thanks for the clarification.